scripts/create_and_deploy_recipe.sh

```diff
@@ -97,17 +97,26 @@ if [[ "${NETWORK}" = "" ]]; then
     NETWORK=${CHANNEL}
 fi
 
+# Build and Deploy binaries
+if [[ ${SKIP_BUILD_DEPLOY} != "true" ]]; then
```

Comment:
this one has to be tighted to the netgoal execution, since the net goal execution would not be there unless you've built it first.

Reply:
included check for existing netgoal binary.